### ISSUE TITLE
Epiphanius and Pseudo-Epiphanius

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#1837]
@SophiaD-M knows this probably best: Does having the two recods PRS7974PseudoE and PRS3828Epiphani make sense or is only one needed?

**Comment by SophiaD-M** [comment]
Yes, I would think it makes sense to differentiate between these two. 
However, the Aksimaros (https://betamasaheft.eu/works/LIT1046Aksima/main) should in that case also be attributed to Pseudo-Epiphanius.
Thank you

**Comment by thea-m** [comment]
Thank you! Should any references to Epiphanius remain or do we mean Pseudo-Epiphanius by all of them? You can see the records in which both are referenced when clicking on the "show attestations" button.
